Quickstart obj-c multistep

cocojoe opened the pull request:
Maintenance update to PR #2328
Reworked Intro/Login inline with current guides
Code Tweaks
Formatting
Copy Tweaks
Fixed broken links
Fixed Project Specs

mpaktiti committed:
Initial commit from existing PRs

mpaktiti committed:
Decapitalize files

mpaktiti committed:
Remove invalid include

mpaktiti committed:
Fix seed include snippet

mpaktiti committed:
Fix title and seed include snippet

mpaktiti committed:
Fix seed download snippets

mpaktiti committed:
Replace uiURL with manage_url

cocojoe committed:
Reworked Intro/Login inline with current guides
Code Tweaks
Formatting
Copy Tweaks
Fixed broken links
Fixed Project Specs

jeffreylees commented on articles/native-platforms/ios-objc/00-introduction.md:
```diff
@@ -0,0 +1,130 @@
+
+The tutorial guides contained in this section will teach you the different things you can do with it. It's recommendable that you follow them sequentially in order, as they are progressive and there is content that usually needs to be learned from previous tutorials.
+
+Said that, we suggest you to start with the [login tutorial](01-login.md)...
```
Should be "That said," rather than "Said that," or you could remove and start with "We suggest...".

Also in the paragraph above this, I'd probably use "It's recommended" - just sounds better to me, although there's nothing wrong with how it is.

jeffreylees commented on articles/native-platforms/ios-objc/02-custom-login.md:
```diff
@@ -158,25 +156,23 @@ You can use the `profile.name` to show the user's name or show the users profile
+                            // Something went wrong, let the user know
+                        } else {
+                            // You signed up correctly, and you have the Credentials,
+                            // so no need to make the user login. He's already in.
```
We try to stay gender neutral for users, so probably "They are already in" was accurate, or "The user is already in". Or just remove that last line "He's already in."

jeffreylees commented on articles/native-platforms/ios-objc/10-customizing-lock.md:
````diff
@@ -0,0 +1,192 @@
+
+You will need to customize several parts. This is how you do it:
+
+```objc
````
It might be better to use an ordered list here and just indent the code blocks to stay inline under their list item, in place of the commented numbering inside each block.

jeffreylees reviewed (commented):
Just a few things to be copyedited that I noticed, nothing big.

cocojoe committed:
Copy Tweaks

cocojoe committed:
Update inline with recent structural changes
Fix relative links
Improve code formatting for readability

cocojoe committed:
Fixed Chapter Links, Fixed iOS Lock v1 Library Links
Spelling Fixes
Remove Carthage
Expand Login section, Safari Auth, Native
Epxand Custom Login, Using WebAuth

cocojoe committed:
Update language naming
